File: Assets/Scripts/Utils/KLog.cs
using UnityEngine;

// kitauji log
public class KLog
{
    public static void I(string TAG, string message)
    {
        Debug.Log("[Info][" + TAG + "]" + " " + message);
    }

    public static void W(string TAG, string message)
    {
        Debug.LogWarning("[Warning][" + TAG + "]" + " " + message);
    }

    public static void E(string TAG, string message)
    {
        Debug.LogError("[Error][" + TAG + "]" + " " + message);
    }
}

File: Server/src/util/k_log.js
export class KLog
{
    static I(TAG, logStr)
    {
        console.log(`[${TAG}] ${logStr}`);
    }

    static W(TAG, logStr)
    {
        console.warn(`[${TAG}] ${logStr}`);
    }

    static E(TAG, logStr)
    {
        console.error(`[${TAG}] ${logStr}`);
    }
}
